src/main/java/org/fogbowcloud/arrebol/execution/k8s/constants/K8sConstants.java

```diff
@@ -0,0 +1,9 @@
+package org.fogbowcloud.arrebol.execution.k8s.constants;
+
+public class K8sConstants {
+	public static final String JSON_CLUSTER_CAPACITY_KEY = "capacity";
+	public static final String JSON_CLUSTER_ADDRESS_KEY = "address";
```

Comment:
it is get from a general conf file? If so, I guess you could rename this properties to something more expressive like: "k8s_cluster_address" or something like that, what do you think?

Reply:
I'm assuming that these props lives in a "arrebol.conf" file. If that's not the case, do not consider my suggestion.

Reply:
Done